Bottom: 3a0e9ef4c6aab64612aefa5ac0b2fefafb8b98c9
Top:    3b8bcc960b1444b2b32583afe07f0add1e0b14a7
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-02-08 21:15:18 +0000

Refresh of instrument-polygon-slowness

---

diff --git a/src/misc.c b/src/misc.c
index c3c2771..4a05b7e 100644
--- a/src/misc.c
+++ b/src/misc.c
@@ -245,10 +245,6 @@ void
 SetPolygonBoundingBox (PolygonTypePtr Polygon)
 {
   PLINE *outer = Polygon->Clipped->contours;
-  if (outer == NULL) {
-    printf ("Clipped->contours was null, sorry\n");
-    return;
-  }
   Polygon->BoundingBox.X1 = outer->xmin;
   Polygon->BoundingBox.Y1 = outer->ymin;
   Polygon->BoundingBox.X2 = outer->xmax;
